v2/js/components/SugarL10n.js

```diff
@@ -0,0 +1,157 @@
+// Localization component
+const SugarL10n = {
+	name: 'SugarL10n',
+	template: '<div/>',
+	data: function () {
+		return {
+			l10n: null,
+			code: null,
+			dictionary: null,
+			eventReceived: false,
+			language: null,
+			units: [
+				{ name: 'Years', factor: 356 * 24 * 60 * 60 },
+				{ name: 'Months', factor: 30 * 24 * 60 * 60 },
+				{ name: 'Weeks', factor: 7 * 24 * 60 * 60 },
+				{ name: 'Days', factor: 24 * 60 * 60 },
+				{ name: 'Hours', factor: 60 * 60 },
+				{ name: 'Minutes', factor: 60 }
+			],
+		};
+	},
+
+	computed: {
+		readyToEmit: function () {
+			return this.dictionary != null;
+		},
+	},
+
+	watch: {
+		readyToEmit: function (newVal, oldVal) {
+			try {
+				if (newVal) {
+					this.$emit('localized');
+					this.eventReceived = true;
+				}
+			} catch (error) {
+				console.log(error);
+			}
+		},
+	},
+
+	mounted: function () {
+
+		const vm = this;
+		if (vm.l10n == null) {
+			vm.language = (typeof chrome != 'undefined' && chrome.app && chrome.app.runtime) ? chrome.i18n.getUILanguage() : navigator.language;
+
+			const language = vm.language;
```

Comment:
Should be replaced by `const language = vm.language.substr(0, 2)` to ensure the two-letter codes language is used.

Reply:
if selected language is suppose igbo then language code for the same will be "ibo "but due to this function it will be resulted to "ib"? should I replace it with language.split("-")[0] ?